ui/src/components/statistics-card.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { Link, useParams } from '@tanstack/react-router';
+
+import {
+  AdvisorJob,
+  AnalyzerJob,
+  EvaluatorJob,
+  ReporterJob,
+  ScannerJob,
+} from '@/api/requests';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+
+type StatisticsCardProps = {
+  job?:
+    | AnalyzerJob
+    | AdvisorJob
+    | ScannerJob
+    | EvaluatorJob
+    | ReporterJob
+    | undefined;
+  title: string;
+  icon?: React.ComponentType<{ className?: string }>;
+  value?: number | string;
+  resultsPath: 'issues' | 'packages' | 'vulnerabilities' | 'rule-violations';
+  pollInterval: number;
+};
+
+export const StatisticsCard = ({
+  title,
+  icon: Icon,
+  value,
+  resultsPath,
+}: StatisticsCardProps) => {
+  const { orgId, productId, repoId, runIndex } = useParams({ strict: false });
```

Comment:
Nor should it use useParams, to allow using it inside any route.

Reply:
You mean it should just take in the `to` as an absolute path?

Reply:
I think it should be a generic card that may or may not be a link, so I'd move the link outside of the component.

Reply:
I want to keep the link inside the component for now, to keep the DOM consistent; otherwise, also Card needs to be taken out of the component to retain the hovering action for link and non-link cards.  

If we need non-link statistics cards like this later on, we could change this.

Reply:
> I want to keep the link inside the component for now, to keep the DOM consistent; otherwise, also Card needs to be taken out of the component to retain the hovering action for link and non-link cards.

You can take in `className` as props like [other components](https://github.com/eclipse-apoapsis/ort-server/blob/main/ui/src/components/ui/card.tsx#L14-L27) do, and then use the `hover:bg-color` at call site.

Reply:
> I want to keep the link inside the component for now, to keep the DOM consistent; otherwise, also Card needs to be taken out of the component to retain the hovering action for link and non-link cards.

You can take in `className` as props like [other components](https://github.com/eclipse-apoapsis/ort-server/blob/main/ui/src/components/ui/card.tsx#L14-L27) do, and then use the `hover:bg-color` at call site.